Compile failure with qt-5.15

mumble-1.3.1_rc1 fails to compile with 5.15 returning error:
```
../../release/.moc/mumble/../../../src/mumble/OverlayText.h:42:16: error: field ‘qpp’ has incomplete type ‘QPainterPath’
   42 |   QPainterPath qpp;
      |                ^~~
```
I also reported bug on gentoo bug tracker with a patch to solve issue https://bugs.gentoo.org/726692

**Desktop (please complete the following information):**
 - OS: gentoo linux
 - Version: mumble-1.3.1_rc1

--- a/src/mumble/OverlayText.h
+++ b/src/mumble/OverlayText.h
@@ -6,6 +6,7 @@
 #ifndef MUMBLE_MUMBLE_OVERLAYTEXT_H_
 #define MUMBLE_MUMBLE_OVERLAYTEXT_H_
 
+#include <QtGui/QPainterPath>
 #include <QtGui/QPixmap>
 #include <QtGui/QFont>
 

